Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls.WebView/WebViewCompatible.cs

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using System.Windows.Forms;
+using Microsoft.Toolkit.UI.Controls;
+using Microsoft.Toolkit.Win32.UI.Controls;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Forms.UI.Controls
+{
+    // [Designer(typeof(WebViewCompatibleDesigner))]
+    [DefaultProperty(Constants.ComponentDefaultProperty)]
+    [DefaultEvent(Constants.ComponentDefaultEvent)]
+    [Docking(DockingBehavior.AutoDock)]
+    [Description("Embeds a view into your application that renders web content using the Microsoft Edge rendering engine")]
```

Comment:
The description is not accurate: Edge MAY be used if it is supported in the environment

Reply:
removed Microsoft Edge, replaced with 'a rendering engine'